.github/workflows/build.yaml

```diff
@@ -0,0 +1,26 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+    name: Build Project
```

Comment:
will `run tests` better?

Reply:
Yes, after all, build is part of the test here in this case